checkboxes/src/main/res/values/styles-checkboxes.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <style name="Widget.Vitamin.Checkbox" parent="Widget.MaterialComponents.CompoundButton.CheckBox">
+        <item name="android:textColor">@color/vtmn_checkbox_text</item>
+        <item name="buttonTint">@color/vtmn_checkbox_box</item>
```

Comment:
Maybe add the text appearance too (body2)? If there is any change in material library, we'll stay with the expected text appearance in our design system.

Reply:
Done ✅ 

I also add the disabled checked state (you can see it in the screenshots)